libs/core/langchain_core/example_selectors/semantic_similarity.py

```diff
@@ -28,6 +28,8 @@ class SemanticSimilarityExampleSelector(BaseExampleSelector, BaseModel):
     input_keys: Optional[List[str]] = None
     """Optional keys to filter input to. If provided, the search is based on
     the input variables instead of all variables."""
+    vectorstore_kwargs: Optional[Dict[str, Any]]
```

Comment:
Lets make this default to None (backwards compatible)

Reply:
ops, I missed that